source/mir/random/discrete.d

```diff
@@ -23,17 +238,17 @@ Params:
 Returns:
     A $(LREF Discrete) sampler that can be called to sample from the distribution.
 */
-Discrete!T discrete(T)(const(T)[] cdPoints)
+NaiveDiscrete!T naiveDiscrete(T)(const(T)[] cdPoints)
```

Comment:
should I adapt `naiveDiscrete` to support an alternative probability-based interface, s.t. the API between both methods is similar?